packages/gatsby/cache-dir/production-app.js

```diff
@@ -106,7 +108,9 @@ apiRunnerAsync(`onClientEntry`).then(() => {
     pagePath &&
     __BASE_PATH__ + pagePath !== browserLoc.pathname &&
     !(
-      loader.findMatchPath(stripPrefix(browserLoc.pathname, __BASE_PATH__)) ||
+      globalLoader.findMatchPath(
```

Comment:
I get an error, it seems `findMatchPath` is not present.

Reply:
Fixed in here: https://github.com/gatsbyjs/gatsby/pull/16334